src/main/java/com/conveyal/gtfs/loader/JdbcGtfsExporter.java

```diff
@@ -153,7 +159,7 @@ public FeedLoadResult exportTables() {
                                 calendarDate.date = date;
                                 calendarDate.service_id = cal.service_id;
                                 calendarDate.exception_type = ex.serviceRunsOn(cal) ? 1 : 2;
-                                LOG.info("Adding exception {} (type={}) for calendar {} on date {}", ex.name, calendarDate.exception_type, cal.service_id, date.toString());
+                                LOG.info("Adding exception {} (type={}) for calendar {} on date {}", ex.name, calendarDate.exception_type, cal.service_id, date);
```

Comment:
Is date now a string?

Reply:
using `toString` is just redundant according to SonarLint. 
<img width="419" alt="image" src="https://github.com/conveyal/gtfs-lib/assets/63798641/20c1c736-9f44-4251-a275-60a67d888d09">